src/components/button/button.tsx

```diff
@@ -45,7 +47,7 @@ export class Button {
      * Set icon for the button
      */
     @Prop({ reflect: true })
-    public icon: string;
+    public icon: string | Omit<Icon, 'backgroundColor'>;
```

Comment:
This seems unnecessarily annoying… 🤔 I think this means that if you have a variable or property typed to `Icon`, you can't pass that into this property without either casting it, or proving to the typescript compiler that the object doesn't contain a property called `backgroundColor`.

Also, this means that rather than the docs looking like this:
<img width="602" alt="image" src="https://github.com/user-attachments/assets/3e15df6c-1cc1-4f4f-b7b4-965bab228624" />

…they look like this:
<img width="602" alt="image" src="https://github.com/user-attachments/assets/db157b20-9030-49f5-ba1f-d893fb9eab2b" />

I would suggest that we instead document that any `backgroundColor` value will be ignored.

Reply:
Aha. Ok. Makes sense. I intentionally didn't make docs about how the iconColor can be used. I have some ideas regarding that, and generally don't want to showcase too much coloring possibilities. So we can perhaps for now not document anything about colors, and not use `backgroundColor` in the code neither. What do you think?

Reply:
⚡